src/lematerial_forgebench/models/registry.py

```diff
@@ -275,15 +275,15 @@ def get_model_config(self, model_name: str) -> Optional[ModelConfig]:
 _registry = ModelRegistry()
 
 
-def get_calculator(model_type: str, **kwargs) -> BaseMLIPCalculator:
+def get_calculator(model_name: str, **kwargs) -> BaseMLIPCalculator:
     """Get a calculator for the specified model type.
```

Comment:
You can see which models are available in your current version of 
        ``forgebench`` using
        :func:`~lematerial_forgebench.models.registry.get_available_models`